services/gateway/config.js

```diff
@@ -111,4 +111,8 @@ config.job = {
 	},
 };
 
+config.cors = {
+	allowedOrigin: process.env.CORS_ALLOWED_ORIGIN || '*',
```

Comment:
Please add description of env variable over here:
https://github.com/LiskHQ/lisk-service/blob/development/services/gateway/README.md

Reply:
Applied.